code/modules/antagonists/clock_cult/scriptures/_clockwork_scripture.dm

```diff
@@ -297,7 +297,7 @@
 	return ..()
 
 /datum/action/innate/clockcult/transmit/Activate()
-	hierophant_message(stripped_input(owner, "What do you want to tell your allies?", "Hierophant Transmit", ""), owner, "<span class='brass'>")
+	hierophant_message(tgui_input_text(owner, "What do you want to tell your allies?", "Hierophant Transmit", ""), owner, "<span class='brass'>", use_sanitisation = FALSE)
```

Comment:
`use_sanitisation = FALSE` is fine because `tgui_input_text` sanitizes by default anyways.

Reply:
I recommend putting a comment then